build/ubuntu/Dockerfile.builder

```diff
@@ -21,6 +21,7 @@ RUN set -x && \
     stack build haskell-src-exts && \
     stack build --pedantic --test --no-run-tests --bench --no-run-benchmarks --dependencies-only -j${THREADS} && \
     stack install ormolu && \
+    cabal build all --dependencies-only -j2 && \
```

Comment:
`-j2` probably not needed, but wtihout it my 8GB machine ran out of memory

Reply:
It might be better to extract a `BUILD_ARG` for this. I think on the build machines it might be best to just add `-j`.

Reply:
Done in https://github.com/wireapp/wire-server/pull/2060/commits/891dd2525061e36d8e648e27417bc22839b6e89d